tika/src/main/java/fr/pilato/elasticsearch/crawler/fs/tika/TikaInstance.java

```diff
@@ -81,63 +83,75 @@ private static void initTika(Fs fs) {
 
     private static void initParser(Fs fs) {
         if (parser == null) {
-            PDFParser pdfParser = new PDFParser();
-            DefaultParser defaultParser;
-            TesseractOCRParser ocrParser;
-
-            // To solve https://issues.apache.org/jira/browse/TIKA-3364
-            // PDF content might be extracted multiple times.
-            pdfParser.getPDFParserConfig().setExtractBookmarksText(false);
-
-             if (ocrActivated) {
-                logger.debug("OCR is activated.");
-                ocrParser = new TesseractOCRParser();
-                if (fs.getOcr().getPath() != null) {
-                    logger.debug("Tesseract Path set to [{}].", fs.getOcr().getPath());
-                    ocrParser.setTesseractPath(fs.getOcr().getPath());
-                }
-                if (fs.getOcr().getDataPath() != null) {
-                    logger.debug("Tesseract Data Path set to [{}].", fs.getOcr().getDataPath());
-                    ocrParser.setTessdataPath(fs.getOcr().getDataPath());
-                }
+            if (fs.getTikaConfigPath() != null && (new File(fs.getTikaConfigPath())).exists()) {
```

Comment:
I think that we should fail early if this condition `new File(fs.getTikaConfigPath())).exists()` is not met.
And raise an exception.